### ISSUE TITLE
Arrow keys don't work very well (at all) for navigation and changing status

### ISSUE THREAD

**Issue by jzohrab** [jzohrab/lute-v3#170]
When reading the tutorial, click on a term and try repeatedly hitting the status update key (arrow up/down).  You can currently only click the arrow up (or down) once, after which the up/down arrow no longer works.

My guess is that, since each status change triggers an ajax reload, the term becomes somehow "deselected" and the arrows don't work.

**Comment by jzohrab** [comment]
Back in 3.0.12, you could click a term, and then hit up/down arrow many times in a row to update the status.  That's been lost since the full page ajax was added.

(Back in 3.0.6, you used to be able to use the up/down arrows to change the status just on hover.  In 3.0.7 that was changed to only allow update for clicked terms.)

**Comment by jzohrab** [comment]
Calling the js method reload_text_div() calls

```
  const url = `/read/renderpage/${bookid}/${pagenum}`;
  const repel = $('#thetext');
  repel.load(url);
```

`renderpage` uses template `    return render_template("read/page_content.html", paragraphs=paragraphs)` which has `  parent.start_hover_mode(false);`, so that automatically starts hovering.

console messages have

```
xxx enter reload_text_div
add kwordmarked to ID-196-1    <<< This is where I try to set the class
mode:
click
in page_content.html
xxx enter start_hover_mode      <<< but this would clear it.
```

So the load of page_content calls script which gets executed ... whenever it gets executed, it happens to occur after I try to take control.  That feels like the main issue.

**Comment by jzohrab** [comment]
That renderpage route is called in only a few places:

```
lute/static/js/lute.js:  const url = `/read/renderpage/${bookid}/${pagenum}`;
lute/templates/read/index.html:    const url = `/read/renderpage/${bookid}/${actualPage}`;
lute/templates/read/updated.html:        const url = `/read/renderpage/${bookid}/${pagenum}`;
```

**Comment by jzohrab** [comment]
Fixed in #179.
